packages/gqltest/gqltest.js

```diff
@@ -142,19 +155,11 @@ function assertExpected(response, expected, label) {
   expected = optionalJSONFromFile(expected, label);
 
   // (2),(3) - Response at the root.
-  if (Object.hasOwn(expected, "data")) {
-    if  (Object.hasOwn(expected, "errors")) {
-      chai.expect.fail("field errors in response not yet supported.")
-    }
+  if (Object.hasOwn(expected, "data") || Object.hasOwn(expected, "errors")) {
     chai.expect(response.body).to.deep.equal(expected);
```

Comment:
Generically this will not work for expected with errors, which is why it needs to be separated.
This is because the order of errors cannot be guaranteed. Expected field errors need to be explicitly tested that a matching error with the expected path exists. Thus need to test that data matches and then process expected errors.